### PR TITLE
feat(dev): handle skip ci label to skip tests

### PR DESCRIPTION
**Pull request by dannycoates**
Set this label if you want to skip ci tests on a PR. Add `fxa-` labels on top to skip specific packages.

### CODE REVIEW

File: .circleci/modules-to-test.js [hunk 1]
```diff
@@ -39,6 +46,31 @@ async function getModules(org, repo, prNumber) {
...
+async function modulesToSkip(org, repo, prNumber, branch) {
+  try {
+    const labelRes = await fetch(
+      `https://api.github.com/repos/${org}/${repo}/issues/${prNumber}/labels`
+    );
+    const labels = await labelRes.json();
+    const skipModules = branch.startsWith('email-service')
+      ? []
+      : ['fxa-email-service'];
+    if (Array.isArray(labels) && labels.find((l) => l.name === '🙈 skip ci')) {
```
Comment (6a68) [inline .circleci/modules-to-test.js:58]: 🙈 😆

**Comment by jodyheavener** [comment]
Can we add some documentation for this? Even a little note in the Contributing doc would be helpful

File: .circleci/modules-to-test.js (hunk 1 above)
Comment (dannycoates) [reply .circleci/modules-to-test.js:58]: I thought the emoji might sort to the top of the label list, but no. still worth keeping though :)

**Comment by clouserw** [comment]
I added what I think the label would be to Jira, but afterwards I realize we don't sync PRs to jira anyway, so, we probably won't see it there.

**Comment by jodyheavener** [comment]
Thanks @dannycoates :D

**Pull request merged**